packages/taquito/src/operations/transfer-ticket-operation.ts

```diff
@@ -0,0 +1,56 @@
+import { OperationContentsAndResult, OperationContentsAndResultTransferTicket, OpKind } from '@taquito/rpc';
+import { Context } from '../context';
+import { Operation } from '../operations';
+import { GasConsumingOperation, StorageConsumingOperation, FeeConsumingOperation, ForgedBytes, RPCTransferTicketOperation } from './types';
+
+/**
+ *
+ * @description TransferTicket operation provides utility function to transfer tickets from an L2 smart contract address to an L1 smart contract address
```

Comment:
Transfer tickets from a Tezos address (tz1,tz2 or tz3) to a smart contract address( KT1) (everything on layer 1 at this step)